cluster/gce/gci/configure-helper.sh

```diff
@@ -900,6 +900,9 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+  # Load existing ABAC policy files written by versions < 1.6 of this script
+  # TODO: only default to this legacy path when in upgrade mode
+  ABAC_AUTHZ_FILE="${ABAC_AUTHZ_FILE:-/etc/srv/kubernetes/abac-authz-policy.jsonl}"
```

Comment:
The container-linux script didn't exist in 1.5.  Is it possible for someone to have an existing install that they upgrade using that script?